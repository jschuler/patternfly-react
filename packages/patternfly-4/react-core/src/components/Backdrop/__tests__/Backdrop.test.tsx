// Auto-generated do not edit

import * as React from 'react';
import { mount } from 'enzyme';
import { Backdrop } from '../Backdrop';

describe('Backdrop test', () => {
  it('Backdrop should match snapshot', () => {
    const view = mount(
      <Backdrop
        children={<>ReactNode</>}
				className={"''"}
      />);
    expect(view).toMatchSnapshot();
  });
});
