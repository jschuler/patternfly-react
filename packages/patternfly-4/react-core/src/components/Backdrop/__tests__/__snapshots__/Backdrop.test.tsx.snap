// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Backdrop test Backdrop should match snapshot 1`] = `
<Component
  className="''"
>
  <div
    className="pf-c-backdrop ''"
  >
    ReactNode
  </div>
</Component>
`;
